<ui version="4.0" >
 <class>QgsZonalStatisticsDialogBase</class>
 <widget class="QDialog" name="QgsZonalStatisticsDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="mRasterLayerLabel" >
     <property name="text" >
      <string>Raster layer:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QComboBox" name="mRasterLayerComboBox" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="mVectorLayerLabel" >
     <property name="text" >
      <string>Polygon layer containing the zones:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QComboBox" name="mPolygonLayerComboBox" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="mColumnPrefixLabel" >
     <property name="text" >
      <string>Output column prefix:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLineEdit" name="mColumnPrefixLineEdit" />
   </item>
   <item row="6" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsZonalStatisticsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsZonalStatisticsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
